docs/modules/ROOT/pages/reference/extensions/core.adoc

```diff
@@ -168,7 +168,7 @@ What to do if it is not possible to extract CSimple expressions from a route def
 
 |icon:lock[title=Fixed at build time] [[quarkus.camel.main.enabled]]`link:#quarkus.camel.main.enabled[quarkus.camel.main.enabled]`
 
-If `true` all `camel-main` features are enabled; otherwise no `camel-main` features are enabled. See described the xref:user-guide/bootstrap.adoc[Bootstrap] camel-main section of the documentation for more details.
+If `true` all `camel-main` features are enabled; otherwise no `camel-main` features are enabled. See described the link:https://camel.apache.org/camel-quarkus/latest/user-guide/bootstrap.html#_camel_main[Bootstrap] section of Camel Quarkus documentation for more details.
```

Comment:
This change probably wasn't intended. A fix was made directly to the generated doc file:

https://github.com/apache/camel-quarkus/commit/c03d38234694fe04eb91bc04de4332fc93bcea8d#diff-46b42b4229cd7a39c564e780bb665a8bde4fdf722007e8473f167fe53ed4b995

So it'll need applying to the doc sources too.

Reply:
I see, let's remove this part from this PR.